README.md

```diff
@@ -131,12 +143,12 @@ Name | Official | Tested | Compatible | Patched
 
 ### Social/communication
 
-Name    | Official | Tested   | Compatible | Patched
---------|----------|----------|------------|----------
-Discord | Dropped  | 1.0.9032 |            |
-Caprine | Broken   | 2.57.1   |            | 🇵 2.59.3
-
-- [Caprine-ng](https://github.com/Alex313031/caprine-ng)
+Name       | Official | Tested   | Compatible | Patched
+-----------|----------|----------|------------|----------
+Discord    | Dropped  | 1.0.9032 |            |
+Caprine    | Broken   | 2.57.1   |            | 🇵 2.59.3
+Caprine-ng | 2.60.1   |
```

Comment:
`Caprine-ng` is already included in the `Patched` column of `Caprine`, no need to add it separately :smiley: 

```suggestion
Caprine    | Broken   | 2.57.1   |            | 🇵 2.60.1 (Caprine-ng)
```